[#72752] 17.2 Project overview page updates  https://community.openpr… (#22243)
opf/openproject: docs/user-guide/project-home/project-widgets/README.md

![Work packages table widget on a project home page in OpenProject](openproject_user_guide_project_home_wp_table_widget.png)

## Budgets widgets

> [!NOTE]

> > Budgets widgets are only available on the **Overview tab** of a project home page.

Reviewer: ```suggestion
> Budgets widgets are currently only available on the **Overview tab** of a project home page.
```

I guess this may change in the future.

Reviewer 2: It is not planned tbh..

Budgets widgets provide a quick overview of the financial status of a project. They summarize planned budgets, actual costs, and remaining budget based on the budgets defined in the project.

![Example of budgets widgets under Overview tab on project home page in OpenProject](openproject_user_guide_project_overview_budgets_widgets.png)

### Prerequisites

Budgets widgets are only displayed if the following conditions are met:
- Both **Budgets** and **Time and costs** modules are activated in the project.

> - At least one budget is created in a project or subproject.

Reviewer: A budget also needs to have planned unit costs and planned labor costs in order to show Budgets by Cost Type (and possibly the other widgets - I need to double check!). **Add a budget with base amount does not suffice!**

Reviewer: https://community.openproject.org/documents/180 was meant to document this. Unfortunately, there has been a bit of churn with this work - so I will do another check.

> - At least one budget is created in a project or subproject.

Author: ```suggestion
- At least one budget exists in the project or one of its subprojects. Some widgets may display placeholder messages if no cost data has been entered yet.
```

> - At least one work package is assigned to a budget.

Reviewer: ```suggestion
- At least one budget is assigned to a work package.
```

@klaustopher could you check my suggestion?

Reviewer 3: I think that's splitting hairs. While we do assign a budget in the work package page, both wordings make sense and the reader understands what is needed. Personally I like the original better than the suggestion, but both are fine in my eyes.

- The user has these necessary permissions to view the financial information:
  - view_budgets
  - view_cost_entries
  - view_cost_rates
  - view_time_entries
  - view_hourly_rates

> - The user has to have a defined an hourly rate.

Author: ```suggestion
- The user has to have a defined hourly rate.
```

### Aggregation across project hierarchy

Budget data is aggregated across the project hierarchy. Budgets from the hierarchy are counted **only once**

If a project has sub-projects, the widgets display the **aggregated values of budgets from the current project and its sub-projects**, provided the user has permission to view those projects and their financial data.

### Budget summary widgets

Four summary widgets provide a quick overview of the financial situation of the project:

- **Total actual costs** – the sum of recorded labor and unit costs
- **Total planned budget** – the total planned amount from all budgets
- **Spent budget** – the percentage of the planned budget that has already been spent
- **Remaining budget** – the difference between planned budget and actual costs

### Budget by cost type

The **Budget by cost type** widget displays a **pie chart** showing the distribution of planned budget across different cost types, including: 
- All cost types currently used in the budgets
- **Labor costs** as an additional cost type (if greater than 0)

> - **Base amount** as an additional cost type (if defined)

Reviewer 2: The base amount is not shown in the charts, but only in the 4 small widgets if I recall correctly @myabc ?

Reviewer: Correct. **Base amount is NOT shown**. This line should be removed.

> - **Base amount** as an additional cost type (if defined)

Author: ```suggestion
```
